src/component/Mount/Mount.js

```diff
@@ -0,0 +1,49 @@
+import { ShowMount } from './ShowMount';
+
+import './mount.scss';
+
+export class Mount extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      counter: 0,
+      color: '',
+      show: true
+    };
+  }
+
+  addBackgroundColor(counter) {
+    const newcounter = counter + 1;
+    if (newcounter % 3 === 0 && newcounter % 5 === 0) {
+      this.setState({ color: 'cyane' });
+    } else if (newcounter % 3 === 0) {
+      this.setState({ color: 'green' });
+    } else if (newcounter % 5 === 0) {
+      this.setState({ color: 'blue' });
+    } else {
+      this.setState({ color: '' });
+    }
+  }
+
+  render() {
+    const { counter, show, color } = this.state;
+    return (
+      <div className="wrap_mount">
+        <button onClick={() => {
+          this.setState({ counter: counter + 1 });
+          this.addBackgroundColor(this.state.counter);
+        }}
```

Comment:
why don't you do it in one place?
It is good pattern to combine states together
```
...
const counter = this.state.counter + 1;
let color = '';

if (counter % 3 === 0 && counter % 5 === 0) {
color = 'cyane';
} else ...
...

this.setState({ counter: counter, color });
```